src/index.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import { render } from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
it ain't been used here

Reply:
take it away

Reply:
That line imports `BrowserRouter` and is given an alias `Router` which is used to wrap the `App` component.
```
<Router>
    <App />
 </Router>
```
Do you mean we should remove the alias and  have something like
```
<BrowserRouter>
    <App />
 </BrowserRouter>
```

Reply:
right!